### PR TITLE
Update CeriseScans.mjs

### LINKED ISSUE #6343: [<Cerise Scans>] Connector not working
### Is there an existing issue for this connector?

- [X] I have searched the existing issues

### Did you read the troubleshooting guide ?

- [X] I read it but nothing seems related

### Is the website of the connector working properly ?

- [X] Yes i'm able to see the manga within my standard browser

### Make sure to try Nightly Build

- [X] Yes nightly build didn't fix the issue

### What kind of issue are you encountering

Chapter list isn't refreshing

### Describe the bug

The chapters are not loading and several more recent manga are missing. In recent days there have been several attacks on online sites and scans (DMCA), some scans have made modifications to the site. Cerise announced that they would use a backup site, perhaps they changed the site. I also noticed that the link for some manga has changed.

This is the current website link: https://cerisescan.com/
And this is the link to a manga: https://cerisescan.com/manga/82/
And this is from one chapter: https://cerisescan.com/manga/82/cap-01/

### To Reproduce

1. Name of connector: Cerise Scans
2. Name of manga (if connector update is working): The name of the manga appears, but the new manga are missing
3. Name of chapter (if manga update is working): No chapter appears.
4. Copy any relevant error message that appears in a popup or (advanced) within the console (press F12)
image

### PR DESCRIPTION
**Pull request by EliseuPinkoski**
Fixes https://github.com/manga-download/hakuneko/issues/6343

### CODE REVIEW

**Review by MikeZeDev — CHANGES_REQUESTED** [review]
I see there is no more chapter protector but there are things to be fixed, and the website got novels so keep Madaranovel template.

File: src/web/mjs/connectors/CeriseScans.mjs [hunk 1]
```diff
@@ -1,35 +1,13 @@
-import WordPressMadaraNovel from './templates/WordPressMadaraNovel.mjs';
+import WordPressMadara from './templates/WordPressMadara.mjs';
 
-export default class CeriseScans extends WordPressMadaraNovel {
+export default class Cerisescan extends WordPressMadara {
```
Comment (MikeZeDev) [inline src/web/mjs/connectors/CeriseScans.mjs]: Class name must match file name, there is no reasons to change it.
Also keep MadaraNovel .

File: src/web/mjs/connectors/CeriseScans.mjs [hunk 2]
```diff
@@ -1,35 +1,13 @@
...
 
     constructor() {
         super();
-        super.id = 'cerisescans';
-        super.label = 'Cerise Scans';
-        this.tags = [ 'webtoon', 'portuguese', 'scanlation' ];
+        super.id = 'CeriseScan';
```
Comment (MikeZeDev) [inline src/web/mjs/connectors/CeriseScans.mjs]: Dont change id, NEVER. Also it must stay lowercase.

File: src/web/mjs/connectors/CeriseScans.mjs [hunk 3]
```diff
@@ -1,35 +1,13 @@
-import WordPressMadaraNovel from './templates/WordPressMadaraNovel.mjs';
+import WordPressMadara from './templates/WordPressMadara.mjs';
```
Comment (MikeZeDev) [inline src/web/mjs/connectors/CeriseScans.mjs]: are you sure there is no more novels on the website? 

Edit : https://cerisescan.com/manga/100/cap-01/ 
Keep MadaraNovel

**Comment by EliseuPinkoski** [comment]
mjs is configured for novel and not for comic.

**Comment by MikeZeDev** [comment]
> mjs is configured for novel and not for comic.
> […](#)
> Message ID: ***@***.*** com>

WordPressMadaraNovel handles both.

**Comment by EliseuPinkoski** [comment]
all works that are not new receive this warning
[image: image.png]

Em sex., 13 de out. de 2023 às 04:28, MikeZeDev ***@***.***>
escreveu:

**Comment by MikeZeDev** [comment]
there is no image and there are corrections to do. I dont understand your last message unfortunately.
If you just do what i say, everything will be fine :)

**Comment by EliseuPinkoski** [comment]
This website in question is mine, it has the images and everything is
correct but it doesn't work, it only works in the novels, but everything is
fine, thank you!


Em sex., 13 de out. de 2023 às 17:43, MikeZeDev ***@***.***>
escreveu:

**Comment by MikeZeDev** [comment]
You made some good changes and some changes that will make it not working.

If you want it to work, trust me, and do the requested changes. If you dont do that nothing will be merged and PR will be closed.

Maybe we are lost in translation.

**Review by MikeZeDev — APPROVED** [review]

**Comment by EliseuPinkoski** [comment]
now it's working perfectly,

I appreciate the changes.

Em sáb., 21 de out. de 2023 às 06:29, MikeZeDev ***@***.***>
escreveu:

**Pull request merged**